docs/about/cookies.md

```diff
@@ -18,6 +22,6 @@ Able Player uses the [js-cookie](https://github.com/js-cookie/js-cookie) library
 
 All cookies are stored locally in your browser and are used **only** to maintain your settings between sessions.
 
-### 🚫 No tracking, no profiling
+## No tracking, no profiling
 
 Able Player does **not** use cookies for advertising, analytics, or behavioral tracking. Your privacy is respected, and no information is shared with third parties.
```

Comment:
Cookie statements should indicate how the *site* uses the cookies. So this should be about how wpaccessibility.org uses cookies, not about Able Player.